pkg/validator/validators.go

```diff
@@ -136,9 +156,53 @@ func ingressBackendValidator(req *admissionv1.AdmissionRequest) (*admissionv1.Ad
 		}
 	}
 
+	err := kc.checkDuplicateBackends(ingressBackend)
+	if err != nil {
+		return nil, err
+	}
+
 	return nil, nil
 }
 
+// checkDuplicateBackends checks to see if the backends present in the given
+// ingressBackend conflict with existing ingressBackends
+func (kc *policyValidator) checkDuplicateBackends(candidateIngressBackend *policyv1alpha1.IngressBackend) error {
```

Comment:
We shouldn't need this if we use the API in policy.Controller to lookup an existing IngressBackend matching the same backend?

Reply:
Ah yep, I was duplicating the loop; I see now where `ingressBackendValidator` already loops through the list of backends. The end result is much simpler 👍🏾